classes/order/OrderDetail.php

```diff
@@ -536,25 +536,38 @@ protected function setVirtualProductInformation($product)
      * Check the order status.
```

Comment:
Perhaps we can take this opportunity to update this description?

Reply:
Indeed, descriptions has been updated:
PhpDoc: `Updates product quantity in stock, according to order status.`
Method name: `updateProductQuantityInStock()`